cell_classification/train.py

```diff
@@ -0,0 +1,149 @@
+from cProfile import label
+import os
+import argparse
+import tensorflow as tf
+import toml
+from augmentation_pipeline import prepare_tf_aug, py_aug, get_augmentation_pipeline
+from segmentation_data_prep import parse_dict, feature_description
+from deepcell.model_zoo.panopticnet import PanopticNet
+from tensorflow.keras.optimizers import SGD, Adam
+from deepcell.utils.train_utils import rate_scheduler, get_callbacks, count_gpus
+from deepcell import losses
+from semantic_head import create_semantic_head
+
+
+class Trainer:
+    """Builds, trains and writes validation metrics for models"""
+
+    def __init__(self, params):
+        """Initialize the trainer with the parameters from the config file
+        Args:
+            params (dict): Dictionary of parameters from the config file
+        """
+        self.params = params
+
+    def prep_data(self):
```

Comment:
Is there a `seed` we can set for the data_prep step? We want the dataset splits to be identical for the same input data if we re-run it when testing other model params. The other option would be to do this ahead of time and save the splits.

Reply:
Nevermind, this is already deterministic since it's just taking from the top of record

Reply:
Yep, but we should probably think about more nuanced ways for data splitting a bit later in the project.